environments/gravity.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
+
+from environments import Environment, visualize_rollout
+
+
+class NObjectGravity(Environment):
+
+    """N Object Gravity Atraction System
+
+    Equations of movement are:
+
+        m_i*q'' = G * sum_{i!=j} ( m_i*m_j*(q_j - q_i)/ abs(q_j  - q_i)^3 )
+
+    """
+
+    def __init__(self, mass, gravity_cst, orbit_noise=.01, q=None, p=None):
+        """Contructor for spring system
+
+        Args:
+            mass ([float]): Object masses (kg).
```

Comment:
As above, should the docstring be
```suggestion
            mass (list): List of floats corresponding to object masses (kg).
```